_data-prepper/monitoring.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+title: Monitoring
+nav_order: 33
+---
+
+# Monitoring Data Prepper with metrics
+
+Data Prepper can be monitored with metrics using [Micrometer](https://micrometer.io/). There are two types of metrics: JVM and system metrics and plugin metrics. [Prometheus](https://prometheus.io/) is used as the default metrics backend.
+
+## JVM and system metrics
+
+JVM and system metrics are runtime metrics used to monitor Data Prepper instances. They include metrics for classloaders, memory, garbage collection, threads, and others. For more information, see [JVM and system metrics](https://micrometer.io/docs/ref/jvm). 
+
+### Naming
+
+JVM and system metrics follow predefined names in [Micrometer](https://micrometer.io/docs/concepts#_naming_meters), such as `jvm_classes_loaded` and `jvm_memory_used`. 
+
+### Serving
+
+By default, metrics are served from the **/metrics/sys** endpoint on the Data Prepper server. The format is a text Prometheus scrape. This port can be used for any frontend that accepts Prometheus metrics, such as [Grafana](https://prometheus.io/docs/visualization/grafana/). The configuration can be updated to serve metrics to other registries like Amazon CloudWatch, which does not require or host the endpoint but publishes the metrics directly to CloudWatch.
```

Comment:
this port --> which port??

Also, active voice: The configuration can be updated --> You can configure Prometheus (?) to serve metrics to other registries...

Reply:
This is the Data Prepper server port.

It is configurable and you have the configuration reference here:

https://github.com/opensearch-project/documentation-website/blob/22bac5fd45427db5d09801c221cfa37d3cf6d92f/_data-prepper/configuration/data-prepper-configuration.md?plain=1#L20

The default value is 4900. But again, users can change this.